webpack/webpack.config.cjs

```diff
@@ -175,8 +175,8 @@ module.exports = function(env, argv) {
         },
       },
     },
-    cache: {
-      type: dev ? 'memory' : 'filesystem',
+    cache: dev ? true : {
```

Comment:
What is happening before / after with this change? We want type 'filesystem' it seems like for production, but do we only need to supply `true` to cache for dev?

Reply:
`true` is equal to `cache: { type: 'memory' }`

The main problem on this pr is that when `cache` receive an object and type is set to `memory`, the `name` property should not be set, otherwise webpack would start to complain.

Reply:
I can confirm that without this change, `npm run start` won't run and complains about this.

Reply:
Thanks for the explanation.  It's hard to understand what "fix webpack cache" means with no text and no linked bug.

Reply:
sorry for that!